crates/conjure-cp-core/src/solver/mod.rs

```diff
@@ -154,6 +154,17 @@ pub enum SolverFamily {
     Minion,
 }
 
+impl SolverFamily {
+    pub fn id_str(&self) -> String {
+        match self {
+            SolverFamily::Minion => "minion",
+            SolverFamily::Sat => "sat",
+            SolverFamily::Smt => "smt",
+        }
+        .into()
+    }
+}
+
```

Comment:
As this function always returns a string literal, we could make this const.

With the signature being `pub const fn id_str(&self) -> &'static str`

Reply:
Also, I think this could be named better, as this is not really an id... With the above function signature as_str or as_lowercase_str could work.